test/tic_tac_toe_pair/middleware_test.clj

```diff
@@ -0,0 +1,13 @@
+(ns tic-tac-toe-pair.middleware-test
+  (:require [clojure.test :refer :all]
+            [ring.mock.request :as mock] 
+            [tic-tac-toe-pair.api :refer [app]]
+            [tic-tac-toe-pair.middleware :refer :all]))
+
+(deftest wrap-500-catchall-test 
```

Comment:
@tk-kowal already mentioned this above, and I agree with his comment. It was pretty tough to grok this test since the `/trouble` endpoint was not implementing any TTT-related behavior. I would try mocking a handler for a real route instead of creating routes that are only used in the tests

Reply:
Thanks for reviewing my code, Cyrus! I'm going to address this by changing the test for this middleware so that it hits a real endpoint related to the game, and will also get rid of the unused endpoint.